### ISSUE TITLE
timeKey hardcoded to Time when not using pivotCSV

### ISSUE THREAD

**Issue by Karang** [NateZimmer/svg_plot.js#1]
This https://github.com/NateZimmer/svg_plot.js/blob/master/svg-plot.js#L98 should be:
```js
x.domain([0, d3.max(data, function(d) { return parseFloat(d[args.timeKey]); })]);
```

**Comment by NateZimmer** [comment]
thanks... fixed in 1f4cb760cc39bb6f2251b287ce53a02ef142afa1 . Whole lib could certainly use some TLC 😁

**Comment by Karang** [comment]
Thank you
